addons/hw_drivers/iot_handlers/interfaces/PrinterInterface.py

```diff
@@ -15,6 +15,12 @@ class PrinterInterface(Interface):
 
     def get_devices(self):
         printer_devices = {}
+        for path in printers:
+            printers[path]['identifier'] = path
+            printers[path]['device-make-and-model'] = printers[path]['printer-make-and-model']
+            printers[path]['device-class'] = 'network'
```

Comment:
If a USB printer is detected by the IoT Box, it will be added to Cups.
At the next boot, it will be in `printers` so it will appear as a network printer

Reply:
Now check if 'usb' in 'device-uri'